src/test/scala/cromwell/engine/db/slick/SlickDataAccessSpec.scala

```diff
@@ -655,7 +655,7 @@ class SlickDataAccessSpec extends FlatSpec with Matchers with ScalaFutures {
       (for {
         _ <- dataAccess.createWorkflow(workflowInfo, Nil, Seq(call), localBackend)
         _ <- dataAccess.updateWorkflowState(workflowId, WorkflowRunning)
-        _ <- dataAccess.updateExecutionBackendInfo(workflowId, call, null)
+        _ <- dataAccess.updateExecutionBackendInfo(workflowId, call, None, null)
       } yield ()).failed.futureValue should be(an[IllegalArgumentException])
     }
```

Comment:
how about a scatter test with an assertion that shard backend info updates only hit a single shard?